spec/rubocop/cop/style/align_parameters_spec.rb

```diff
@@ -504,7 +504,9 @@
                              '           b)',
                              'end'])
         expect(cop.offenses.size).to eq 1
-        expect(cop.offenses.first.to_s).to match(/method definition/)
+        expect(cop.offenses.map(&:to_s))
```

Comment:
I'm also not sure why you decided to place the check in this particular example. Seems it should have been done closer to the beginning of the fixed-indent context.